web/src/upload.js

```diff
@@ -302,6 +302,48 @@ export function setup_upload(config) {
     $drag_drop_container.on("dragover", (event) => event.preventDefault());
     $drag_drop_container.on("dragenter", (event) => event.preventDefault());
 
+    $("#main_div").on("dragover", (event) => event.preventDefault());
+    $("#main_div").on("dragenter", (event) => event.preventDefault());
```

Comment:
I think this logic is in the wrong place -- this function gets called more than once, one for each edit input. I think you need to add an `upload.initialize` method called from `ui_init.js` that can do this part of the setup, which is global logic we want to do exactly once.